src/main/java/com/conveyal/datatools/editor/controllers/api/AgencyController.java

```diff
@@ -73,6 +76,8 @@ public static Object createAgency(Request req, Response res) {
         } catch (Exception e) {
             e.printStackTrace();
             halt(400);
+        } finally {
+            if (tx != null) tx.rollbackIfOpen();
```

Comment:
I'm curious why we use rollback in some places (AgencyController) and rollbackIfOpen in other places.